norbit-webapp/components/dashboard/Dashboard.tsx

```diff
@@ -15,45 +13,36 @@ export type TamplifyInstance = typeof Amplify;
 
 const Dashboard: React.FC = () => {
   let sensorConfig = useAppSelector((state) => state.sensorConfig) as SensorConfig;
-  let [amplifyEnabled, setAmplifyEnabled] = useState(false);
-
-  const config = useMemo(() => { 
-    return {
-      identityPoolId: process.env.NEXT_PUBLIC_IDENTITY_POOL_ID,
-      region: process.env.NEXT_PUBLIC_REGION,
-      userPoolId: process.env.NEXT_PUBLIC_USER_POOL_ID,
-      userPoolWebClientId: process.env.NEXT_PUBLIC_USER_POOL_WEB_CLIENT_ID,
-    };
-  }, []);
-
-  const mockAmplify = process.env["NEXT_PUBLIC_MOCK_AMPLIFY"] == "yes";
-
-  useEffect(() => {
-    if (!amplifyEnabled) {
-      if (!mockAmplify) {
-        console.log("Running Amplify in live mode");
+  const mockAmplify = useAppSelector((state) => state.amplify.isMock);
+  const user = useAppSelector((state) => state.amplify.userName);
+  const devices = useAppSelector((state) => state.deviceList.devices);
+  const lastConnectedDevice = devices[devices.length - 1];
+
+
+  if (!lastConnectedDevice) {
+    return (
+      <div className={dashboardStyles.dashboardContainer} id="dashboard-root">
+        <div className={dashboardStyles.chartsContainer} id="dashboard-chart-container">
+          <h1>Connect a device to see data</h1>
+        </div>
+      </div>
+    );
+    
+  }
+  const accelerometerTopic = `${user}/${lastConnectedDevice.code}/accelerometer`;
+  const temperatureTopic = `${user}/${lastConnectedDevice.code}/temperature`;
+  const lightTopic = `${user}/${lastConnectedDevice.code}/lux`;
+  const soundTopic = `${user}/${lastConnectedDevice.code}/volume`;
+  const locationTopic = `${user}/${lastConnectedDevice.code}/location`;
```

Comment:
This is setup this way for now because not have been successfull in extracting identityId in flutter. idealy we would use identityId becasue this is not accessable for others.